src/views/groups/GroupEdit/components/EditGroupMembers.vue

```diff
@@ -0,0 +1,244 @@
+
+              <span v-dompurify-html="htmlText" style="color: white"></span>
+            </div>
+          </template>
+        </vue-bootstrap-typeahead>
+        <div>
+            <b-button
+                class="btn-lg mt-2"
+                type="submit"
+                variant="primary"
+                @click="inviteAdmin"
+                :disabled="isSending || !targetName.length"
+            >
+                {{ $t('activity-feed:send') }}
+                <b-spinner v-if="isSending" small></b-spinner>
+            </b-button>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+</template>
+<script lang="ts">
+  import axios, { AxiosInstance } from 'axios';
+  // @ts-ignore
+  import debounce from 'lodash.debounce';
+  import { Component, Vue, Mixins, Prop, Watch, Ref } from 'vue-property-decorator';
+  import EditField from '@/components/Common/EditField.vue';
+  // @ts-ignore
+  import VueBootstrapTypeahead from 'vue-bootstrap-typeahead';
+
+  @Component({ components: { EditField, VueBootstrapTypeahead } })
+  export default class EditGroupMembers extends Vue {
+    @Prop() readonly parentNID?: number;
+
+    @Prop() readonly uid?: number;
+
+    commentText: string = '';
+
+    targetName = '';
+
+    isSending: boolean = false;
+
+    usernames = [];
+
+    messagesSent = 0;
+
+    async fetchData() {
+      const res = await axios.get(
+        `/get/?type=usernames&size=10${this.targetName ? `&search=${this.targetName}` : ''}`,
+      );
+      this.usernames = res.data.data.usernames;
+    }
+
+    created() {
+      this.fetchData = debounce(this.fetchData, 200);
+    }
+
+    @Watch('targetName', { immediate: true, deep: true })
+    getUserNames() {
+      this.fetchData();
+    }
+
+    @Ref('typeahead') readonly typeahead!: { inputValue: string };
+
+    mounted() {
+      if (this.$route.query.message) {
+        this.typeahead.inputValue = String(this.$route.query.message);
+        this.targetName = String(this.$route.query.message);
+      }
+    }
+
+    setCommentText(text: string) {
+      this.commentText = text;
+    }
+
+    async postMemberInvite(targetUid: string, message: string) {
+      const params = {
+        type: 'invite_member',
+        group_nid: (this.parentNID? this.parentNID : 'null').toString(),
+        action: 'add',
+        target_names: this.targetName,
+      };
+
+      // @ts-ignore
+      if (this.parentNID) params.parent_nid = this.parentNID;
+
+      await axios.post('/post/?type=message', new URLSearchParams(params));
+    }
+
+    async postAdminInvite(targetUid: string, message: string) {
+      const params = {
+        type: 'add_admin',
+        group_nid: (this.parentNID? this.parentNID : 'null').toString(),
+        action: 'add',
+        target_names: this.targetName,
+      };
+
+      // @ts-ignore
+      if (this.parentNID) params.parent_nid = this.parentNID;
+
+      await axios.post('/post/?type=message', new URLSearchParams(params));
+    }
```

Comment:
No parent nid should be sent with either message query - that's only for direct message "threads" (you're passing it the group ID, which is not a valid notification ID).